README.md

```diff
@@ -226,20 +226,15 @@ make
 
 ## Editor/IDE Plugins
 
-To bring IDE functions for the V programming languages to your editor, check out
-[v-analyzer](https://github.com/vlang/v-analyzer). It provides a
-[VS Code extension](https://marketplace.visualstudio.com/items?itemName=VOSCA.vscode-v-analyzer)
-and language server capabilities for other editors.
-
-The plugin for JetBrains IDEs (IntelliJ, CLion, GoLand, etc.) also offers a great development
-experience with V. You can find all features in [its documentation](https://plugins.jetbrains.com/plugin/20287-vlang/docs/syntax-highlighting.html).
+- [VS Code](https://marketplace.visualstudio.com/items?itemName=VOSCA.vscode-v-analyzer)
+- [Vim](https://github.com/vlang/awesome-v#vim)
+- [Emacs](https://github.com/vlang/awesome-v#emacs)
+- [Sublime Text 3](https://github.com/vlang/awesome-v#sublime-text-3)
+- [Atom](https://github.com/vlang/awesome-v#atom)
+- [JetBrains](https://plugins.jetbrains.com/plugin/20287-vlang/docs/syntax-highlighting.html)
```

Comment:
It may be better to sort them in alphabetical order.

Reply:
Definitely, alpha sort is much better for humans.  Many times, your eyes can find something in a short alphabetical list faster than doing a search in the browser.

Reply:
good idea i updated PR